### PR TITLE
IBX-7966: Ibexa Engage described in User Documentation

### PR DESCRIPTION
**Pull request by julitafalcondusza**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | [IBX-8268](https://issues.ibexa.co/browse/IBX-8268)
| Versions      | master
| Edition       | Experience, Commerce

Preview: https://ez-systems-developer-documentation--296.com.readthedocs.build/projects/userguide/en/296/ibexa_engage/ibexa_engage/

Developer PR: https://github.com/ibexa/documentation-developer/pull/2339

#### Checklist

- [x] Text renders correctly
- [x] Text has been checked with vale
- [x] Description metadata is up to date
- [x] Redirects cover removed/moved pages
- [ ] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

**Review by radzyu — CHANGES_REQUESTED** [review]
@julitafalcondusza I've added comments to selected sections.
There is also some general remark related to navigation and order of sections in the User guide.
Together with @SylvainGuittard we think it’s confusing, not consistent with product versions, backoffice menu. Our suggestion:
- PIM should be after Content management (not at the bottom under Commerce and Perso) - firstly you need to create products, to later sell them online using Commerce :) 
- Commerce should be above Perso (Perso is add-on, not core product/capability)
- Customer management and Website organization - @SylvainGuittard where will you put these sections, what's your suggestion?
It could be addressed in a separate ticket/PR.

File: docs/ibexa_engage/ibexa_engage.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+- **Mobile** - uses a link from Qualifio to integrate a campaign that is optimized for mobile usage.
+
+!!! note
+    Each publication channel type is automatically responsive and can be use on any mobile device.
+
+## Work with campaigns
+
+You can create, configure, and create campaigns using [Qualifio Engage](https://support.qualifio.com/hc/en-us/categories/202280638-Qualifio-Engage).
+To display your campaign on your website, use of the available tools - Campaign block or Campaign custom tag.
+
```
[the remark this answers is not in the document]
Comment (julitafalcondusza) [reply docs/ibexa_engage/ibexa_engage.md:44]: In my opinion, it's clear enough.
Please check Ibexa Cloud documentation - here we mention Platform.sh documentation in the same way: https://doc.ibexa.co/en/latest/getting_started/install_on_ibexa_cloud/#install-on-ibexa-cloud

**Comment by julitafalcondusza** [comment]
@radzyu - regarding your comment about navigation/ToC reorganization - this needs to be addressed in separate ticket/PR.

File: docs/content_management/block_reference.md [hunk 2]
```diff
@@ -87,6 +88,25 @@ On the **Properties** tab, set values in the following fields:
 
 On the **Design** tab, in the **View** field, select the layout to be used to present a list of products and submit your changes.
 
+## Campaign block
+
+Displays campaign created and configured in Ibexa Engage.
+On the **Properties** tab, set values in the following fields:
+
+- **Name** – Enter a name for the campaign.
+- **Campaign** – Select campaign to display on the Landing Page.
```
Comment (SylvainGuittard) [inline docs/content_management/block_reference.md:97]: @radzyu can we mention which campaigns are visible in the list?

File: docs/ibexa_engage/ibexa_engage.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+![Welcome page - active account](img/welcome_page_activated.png "Welcome page - active account")
+
+Click **Manage campaigns in Qualifio** button to open Qualifio page, where you can create and manage campaigns.
+
+![Qualifio page](img/qualifio_page.png "Qualifio page - campaigns manager")
+
+## Campaign
+
+A campaign is a collection of ideas that the user can customize and divide into steps.
```
Comment (SylvainGuittard) [inline docs/ibexa_engage/ibexa_engage.md:38]: I am not sure it's the proper definition of a campaign in Qualifio. We need to review it

File: docs/ibexa_engage/ibexa_engage.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+It consists of a welcome screen, a form step, an interaction element, and an exit screen.
```
Comment (SylvainGuittard) [inline docs/ibexa_engage/ibexa_engage.md:40]: It really depends on the types of campaigns. Sometimes you have more parameters

File: docs/ibexa_engage/ibexa_engage.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+
+You can use Qualifio's existing templates and interactive elements, such as quizzes, pools, and forms, to create visually appealing, customized campaigns.
+Qualifio configures the backgrounds, themes, and designs.
```
Comment (SylvainGuittard) [inline docs/ibexa_engage/ibexa_engage.md:43]: What do you mean by `Qualifio`? The team or the product. 
Users can definitely customize design. Sometimes, the Studio team at Qualifio is doing it. I suggest you make it more generic and just mention `Users can configure backgrounds, themes, designs and more`

File: docs/ibexa_engage/ibexa_engage.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+
+Use campaign to interact with viewers, boost ad impressions, or gather cookies from targeted audiences.
+
+Each campaign comes with a unique ID - string of numbers, that the platform automatically defines at the creation. Campaign ID is a reference to the campaign in the applications, services and data connections.
+
+You can set up prizes for customer when an email or another required pool is filled.
```
Comment (SylvainGuittard) [inline docs/ibexa_engage/ibexa_engage.md:49]: Prizes don't apply to all campaigns. I will remove this part and leave it to Qualifio documentation. Same for the line below

Comment (julitafalcondusza) [reply docs/ibexa_engage/ibexa_engage.md:49]: I would keep it (I've just updated it a bit). We say "you can", it doesn't mean that it applies to all campaigns.

File: docs/content_management/block_reference.md (hunk 2 above)
Comment (julitafalcondusza) [reply docs/content_management/block_reference.md:97]: It's mentioned in the "Campaign block" section: _"This list includes all campaigns available on user's Qualifio account which are active or scheduled to launch in the future."_
in my opinion there is no need to go into details here on the Block reference page.

**Comment by radzyu** [comment]
> @radzyu - regarding your comment about navigation/ToC reorganization - this needs to be addressed in separate ticket/PR.

@julitafalcondusza Could you create this ticket and move these remarks/requested changes

File: docs/content_management/block_reference.md (hunk 2 above)
Comment (radzyu) [reply docs/content_management/block_reference.md:97]: @julitafalcondusza I don't see this mention here in the section: https://ez-systems-developer-documentation--296.com.readthedocs.build/projects/userguide/en/296/content_management/block_reference/#campaign-block

File: docs/ibexa_engage/ibexa_engage.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
+---
+description: Ibexa Engage helps to engage audiences and collect valuable data by using interactive content.
+---
+
+# Ibexa Engage
```
Comment (radzyu) [inline docs/ibexa_engage/ibexa_engage.md:5]: I'm just wondering about the name of this section, does it make sense to add prefix "Ibexa" or just Engage? Most sections don't contain it (Perso, Commerce, DAM etc.). :)

File: docs/ibexa_engage/ibexa_engage.md (hunk 4 above)
Comment (radzyu) [reply docs/ibexa_engage/ibexa_engage.md]: @SylvainGuittard it's based on the general definition used by Qualifio: https://developers.qualifio.com/docs/engage/concepts

File: docs/ibexa_engage/ibexa_engage.md (hunk 5 above)
Comment (radzyu) [reply docs/ibexa_engage/ibexa_engage.md]: Agree, here is the entire chapter how users can design campaigns: https://support.qualifio.com/hc/en-us/sections/203320047-Design-campaigns

File: docs/ibexa_engage/ibexa_engage.md (hunk 6 above)
Comment (radzyu) [reply docs/ibexa_engage/ibexa_engage.md]: I think it's more important in the Campaign definition to mention about anonymous vs identified campaign, because it's related to data collection and all campaign types. One sentence could be sufficient.
Source: https://developers.qualifio.com/docs/engage/concepts#anonymous--identified-campaigns

File: docs/ibexa_engage/img/campaign_custom_tag.png
Comment (radzyu) [inline docs/ibexa_engage/img/campaign_custom_tag.png:1]: @julitafalcondusza at this moment, I would not add this screenshot without dropdown and "1" in the Campaign field.
Maybe we can paste here Campaign name (to fake it a bit) or just wait for the final form.

File: docs/ibexa_engage/ibexa_engage.md [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+3\. On the welcome page, provide the required information:
+
+- First and last name
+- Email address
+
+4\. Select the **I have read and agree to the Terms and Conditions** checkbox, and then click **Send request**.
+
+![Welcome page - form](img/welcome_page_form.png "Welcome page - form")
+
+[[= product_name =]] creates and provides user account. An invitation link is sent during the setup process.
```
Comment (radzyu) [inline docs/ibexa_engage/ibexa_engage.md:27]: "Ibexa DXP creates and provides user account. An invitation link is sent during the setup process."-  this sentence sounds for me like the process would be automatic. And it's not at the moment. It's manual and credentials are sent by our team, right @SylvainGuittard ?
Maybe it's a matter of changing the word from Ibexa DXP to Ibexa team?

**Review by radzyu — CHANGES_REQUESTED** [review]
I've added additional comments to selected sentences and screens.

File: docs/ibexa_engage/ibexa_engage.md (hunk 7 above)
Comment (julitafalcondusza) [reply docs/ibexa_engage/ibexa_engage.md:5]: I'd keep Ibexa Engage - even on the welcome page we use _"Welcome to **Ibexa Engage**"_, _"Discover **Ibexa Engage**"_, etc. 
Check Ibexa Cloud or Ibexa CDP - we use full name as well.

File: docs/ibexa_engage/img/campaign_custom_tag.png
Comment (julitafalcondusza) [reply docs/ibexa_engage/img/campaign_custom_tag.png:1]: As discussed - screenshot will be replaced (if ready). If not, will be removed for now.

File: docs/ibexa_engage/ibexa_engage.md (hunk 8 above)
Comment (julitafalcondusza) [reply docs/ibexa_engage/ibexa_engage.md:27]: Changed to _Ibexa team_.

File: docs/ibexa_engage/ibexa_engage.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (radzyu) [reply docs/ibexa_engage/ibexa_engage.md:5]: Ok, so please create a separate ticket.

**Review by radzyu — CHANGES_REQUESTED** [review]
One last comment/change: https://github.com/ibexa/documentation-user/commit/845f59a9efed96ed28ad0b1c722fd430032ae693#r142807053

File: docs/ibexa_engage/img/campaign_custom_tag.png
Comment (juskora) [inline docs/ibexa_engage/img/campaign_custom_tag.png:1]: It would be good to add a frame here to indicate the Engage icon :)

File: docs/ibexa_engage/ibexa_engage.md [hunk 9]
```diff
@@ -0,0 +1,85 @@
...
+
+A campaign is a set of different concepts, grouped per steps, and open to configuration.
+You can use campaign to interact with viewers, boost ad impressions, or gather cookies from targeted audiences.
+
+Campaign can contain a welcome screen, a form step, an interaction element, an exit screen, and so on.
+
+You can use Qualifio's existing templates and interactive elements, such as quizzes, pools, and forms, to create visually appealing, customized campaigns.
+Users can configure the backgrounds, themes, or designs, and set up a specific time frame for each campaign.
+
+Each campaign comes with a unique ID - string of numbers, that the platform automatically defines at the creation. Campaign ID is a reference to the campaign in the applications, services and data connections.
```
Comment (juskora) [inline docs/ibexa_engage/ibexa_engage.md:46]:
```suggestion
Each campaign comes with a unique ID - string of numbers, that the platform automatically defines during the creation. Campaign ID is a reference to the campaign in the applications, services and data connections.
```

Comment (juskora) [reply docs/ibexa_engage/ibexa_engage.md:46]: I'm not sure about creation of what, a campaign or an account?

File: docs/ibexa_engage/ibexa_engage.md [hunk 10]
```diff
@@ -0,0 +1,85 @@
...
+
+Every campaign includes a minimum of one publication channel that you can choose from the three options the platform provides for publishing a campaign:
+
+- **Widget/iframe** - uses a JavaScript code or an HTML iframe from Qualifio. Additionally, you can use oEmbed, Facebook Instant articles, and AMP technologies to publish iframes. You need to manually paste the code into your website or CMS.
+
+- **Minisite** - uses a unique URL from Qualifio that can hosts the campaign. This URL points to your subdomain.
+
+- **Mobile** - uses a link from Qualifio to integrate a campaign that is optimized for mobile usage.
+
+!!! note
```
Comment (juskora) [inline docs/ibexa_engage/ibexa_engage.md:63]: automatically? Or maybe ...by default...?

File: docs/ibexa_engage/ibexa_engage.md [hunk 11]
```diff
@@ -0,0 +1,85 @@
...
+
+You can create, configure, and create campaigns using [Qualifio Engage](https://developers.qualifio.com/docs/engage/).
+To access the Qualifio Back Office page and create or manage campaigns, click the **Manage campaigns in Qualifio** button under the **Ibexa Engage** menu.
+To display your campaign on the website, use one of the available tools - Campaign block or Campaign custom tag.
+For more information about creating and managing campaigns, see [Qualifio Engage documentation](https://support.qualifio.com/hc/en-us/categories/202280638-Qualifio-Engage).
+
+### Campaign block
+
+To promote your campaign, you can add [Campaign block](../content_management/block_reference.md#campaign-block) in Page Builder.
+In the **Campaign** tab you can choose campaign from the drop-down list. This list includes all campaigns available on user's Qualifio account which are active or scheduled to launch in the future.
```
Comment (juskora) [inline docs/ibexa_engage/ibexa_engage.md:76]:
```suggestion
In the **Campaign** block you can choose campaign from the drop-down list. This list includes all campaigns available on user's Qualifio account which are active or scheduled to launch in the future.
```

**Review by juskora — CHANGES_REQUESTED** [review]

**Review by juskora — APPROVED** [review]

File: docs/ibexa_engage/ibexa_engage.md (hunk 11 above)
Comment (radzyu) [reply docs/ibexa_engage/ibexa_engage.md]: Maybe we could add here * , because by default this list contains campaigns "active or scheduled to launch in the future". But it can be changed by devs, if you configure Campaign feed in Qualifio differently. https://support.qualifio.com/hc/en-us/articles/360022954454-About-Campaign-Feeds

**Review by radzyu — APPROVED** [review]

File: docs/ibexa_engage/ibexa_engage.md [hunk 12]
```diff
@@ -0,0 +1,85 @@
...
+
+- First and last name
+- Email address
+
+4\. Accept **Terms and Conditions** checkbox, and then click **Send request**.
+
+![Welcome page - form](img/welcome_page_form.png "Welcome page - form")
+
+[[= product_name_base =]] team creates and provides user account. An invitation link is sent during the setup process.
+Once the registration and configuration process is completed, the welcome page displays information about campaign tools available with [[= product_name_engage =]] and link to [Qualifio documentation](https://support.qualifio.com/hc/en-us/categories/202280638-Campaigns) where you can get information about creating and managing campaigns.
```
Comment (dabrt) [inline docs/ibexa_engage/ibexa_engage.md:28]:
```suggestion
Once the registration and configuration process is completed, the welcome page displays information about campaign tools available with [[= product_name_engage =]] and a link to [Qualifio documentation](https://support.qualifio.com/hc/en-us/categories/202280638-Campaigns) where you can get information about creating and managing campaigns.
```

File: docs/ibexa_engage/ibexa_engage.md [hunk 13]
```diff
@@ -0,0 +1,85 @@
...
+
+![Welcome page - active account](img/welcome_page_activated.png "Welcome page - active account")
+
+Click **Manage campaigns in Qualifio** button to open Qualifio page, where you can create and manage campaigns.
```
Comment (dabrt) [inline docs/ibexa_engage/ibexa_engage.md:32]:
```suggestion
Click the **Manage campaigns in Qualifio** button to open a Qualifio page, where you can create and manage campaigns.
```

Is it Qualifio or Qualifio Manager page?

File: docs/ibexa_engage/ibexa_engage.md [hunk 14]
```diff
@@ -0,0 +1,85 @@
...
+
+![Qualifio Manager](img/qualifio_manager.png "Qualifio Manager - campaign list")
+
+## Campaign
+
+A campaign is a set of different concepts, grouped per steps, and open to configuration.
+You can use campaign to interact with viewers, boost ad impressions, or gather cookies from targeted audiences.
```
Comment (dabrt) [inline docs/ibexa_engage/ibexa_engage.md:39]:
```suggestion
Campaigns are sets of different concepts, grouped per steps, and open to configuration.
You can use a campaign to interact with viewers, boost ad impressions, or gather cookies from targeted audiences.
```
grouped _into_ steps ?

File: docs/ibexa_engage/ibexa_engage.md [hunk 15]
```diff
@@ -0,0 +1,85 @@
...
+
+Campaign can contain a welcome screen, a form step, an interaction element, an exit screen, and so on.
```
Comment (dabrt) [inline docs/ibexa_engage/ibexa_engage.md:41]:
```suggestion
Each campaign can contain a welcome screen, a form step, an interaction element, an exit screen, and so on.
```

File: docs/ibexa_engage/ibexa_engage.md [hunk 16]
```diff
@@ -0,0 +1,85 @@
...
+
+You can use Qualifio's existing templates and interactive elements, such as quizzes, pools, and forms, to create visually appealing, customized campaigns.
+Users can configure the backgrounds, themes, or designs, and set up a specific time frame for each campaign.
```
Comment (dabrt) [inline docs/ibexa_engage/ibexa_engage.md:44]: Why do you say "you" in one sentence and "users" in the other? Is it that "you" is the campaign administrator and "users" are regular editors?

If there is no distinction betweeh "you" and "users", i'd say "You can also configure backgrounds..."

File: docs/ibexa_engage/ibexa_engage.md [hunk 17]
```diff
@@ -0,0 +1,85 @@
...
+
+Each campaign comes with a unique ID - string of numbers, that the platform automatically defines during the creation of campaign. Campaign ID is a reference to the campaign in the applications, services and data connections.
```
Comment (dabrt) [inline docs/ibexa_engage/ibexa_engage.md:46]:
```suggestion
At creation, each campaign is automatically assigned a unique numerical ID. 
Campaign IDs are then used as a reference to the campaign in applications, services and data connections.
```

File: docs/ibexa_engage/ibexa_engage.md [hunk 18]
```diff
@@ -0,0 +1,85 @@
...
+
+There are two types of campaigns:
+
+- **Anonymous** - engages with the audience, increases the advertising impressions, or collects cookies of personalized audiences.
+- **Identified** - gathers information on the user by filling out an identification form.
```
Comment (dabrt) [inline docs/ibexa_engage/ibexa_engage.md:51]:
```suggestion
- **Anonymous** - engage with the audience, increase the advertising impressions, or collect cookies of personalized audiences.
- **Identified** - gather information about the user by filling out an identification form.
```
Who is filling out the form, the user or the campaign process? I'd reword the latter sentence to make it clearer.

File: docs/ibexa_engage/ibexa_engage.md [hunk 19]
```diff
@@ -0,0 +1,85 @@
...
+
+## Publication channel
+
+Every campaign includes a minimum of one publication channel that you can choose from the three options the platform provides for publishing a campaign:
```
Comment (dabrt) [inline docs/ibexa_engage/ibexa_engage.md:55]:
```suggestion
Every campaign includes at least one publication channel, which you can choose from the three options the platform offers:
```

File: docs/ibexa_engage/ibexa_engage.md [hunk 20]
```diff
@@ -0,0 +1,85 @@
...
+
+- **Widget/iframe** - uses a JavaScript code or an HTML iframe from Qualifio. Additionally, you can use oEmbed, Facebook Instant articles, and AMP technologies to publish iframes. You need to manually paste the code into your website or CMS.
+
+- **Minisite** - uses a unique URL from Qualifio that can hosts the campaign. This URL points to your subdomain.
```
Comment (dabrt) [inline docs/ibexa_engage/ibexa_engage.md:59]:
```suggestion
- **Minisite** - uses a unique URL from Qualifio that hosts the campaign. This URL points to your subdomain.
```

File: docs/ibexa_engage/ibexa_engage.md [hunk 21]
```diff
@@ -0,0 +1,85 @@
...
+
+- **Mobile** - uses a link from Qualifio to integrate a campaign that is optimized for mobile usage.
+
+!!! note
+    By default, every type of publication channel is responsive and compatible with all mobile devices.
+
+## Work with campaigns
```
Comment (dabrt) [inline docs/ibexa_engage/ibexa_engage.md:66]: I'd make this a separate article

File: docs/ibexa_engage/ibexa_engage.md [hunk 22]
```diff
@@ -0,0 +1,85 @@
...
+
+You can create, configure, and create campaigns using [Qualifio Engage](https://developers.qualifio.com/docs/engage/).
```
Comment (dabrt) [inline docs/ibexa_engage/ibexa_engage.md:68]:
```suggestion
You can create, configure, and manage campaigns by using [Qualifio Engage](https://developers.qualifio.com/docs/engage/).
```

File: docs/ibexa_engage/ibexa_engage.md [hunk 23]
```diff
@@ -0,0 +1,85 @@
...
+To access the Qualifio Back Office page and create or manage campaigns, click the **Manage campaigns in Qualifio** button under the **Ibexa Engage** menu.
```
Comment (dabrt) [inline docs/ibexa_engage/ibexa_engage.md:69]: Is it "Qualifio Back Office", or could we use the ""Qualifio Manager" term that you introduced above?
Also, I'd try to describe navigation to the button in the following manner: "go to the Back Office, **Ibexa Engage**, and click the **Manage campaigns in Qualifio** button.

File: docs/ibexa_engage/ibexa_engage.md [hunk 24]
```diff
@@ -0,0 +1,85 @@
...
+To display your campaign on the website, use one of the available tools - Campaign block or Campaign custom tag.
```
Comment (dabrt) [inline docs/ibexa_engage/ibexa_engage.md:70]: To display or to embed?

File: docs/ibexa_engage/ibexa_engage.md [hunk 25]
```diff
@@ -0,0 +1,85 @@
...
+For more information about creating and managing campaigns, see [Qualifio Engage documentation](https://support.qualifio.com/hc/en-us/categories/202280638-Qualifio-Engage).
```
Comment (dabrt) [inline docs/ibexa_engage/ibexa_engage.md:71]:
```suggestion


For more information about creating and managing campaigns, see [Qualifio Engage documentation](https://support.qualifio.com/hc/en-us/categories/202280638-Qualifio-Engage).
```

File: docs/ibexa_engage/ibexa_engage.md [hunk 26]
```diff
@@ -0,0 +1,85 @@
...
+
+### Campaign block
+
+To promote your campaign, you can add [Campaign block](../content_management/block_reference.md#campaign-block) in Page Builder.
```
Comment (dabrt) [inline docs/ibexa_engage/ibexa_engage.md:75]:
```suggestion
To promote your campaign, you can add a [Campaign block](../content_management/block_reference.md#campaign-block) to a page in Page Builder.
```

File: docs/ibexa_engage/ibexa_engage.md [hunk 27]
```diff
@@ -0,0 +1,85 @@
...
+In the **Campaign** block you can choose campaign from the drop-down list. This list includes all campaigns available on user's Qualifio account which are active or scheduled to launch in the future.
```
Comment (dabrt) [inline docs/ibexa_engage/ibexa_engage.md:76]:
```suggestion
In the block's settings, you can select a campaign that you want to display from a drop-down list.
The list contains only those campaigns from your Qualifio account, which are active or scheduled to launch in the future.
```
See comment above about account naming

File: docs/ibexa_engage/ibexa_engage.md [hunk 28]
```diff
@@ -0,0 +1,85 @@
...
+
+![Campaign block](img/campaign_block.png "Campaign block")
+
+### Campaign custom tag
+
+You can embed campaign in the Rich Text field with Campaign custom tag.
```
Comment (dabrt) [inline docs/ibexa_engage/ibexa_engage.md:82]:
```suggestion
You can embed a campaign within the [Rich Text field](../content_management/create_edit_content_items.md#edit-rich-text-fields) by using a Campaign custom tag.
```
Question: Should we use "custom tag" here? Editors do not know this concept. For them it is jus one more button in the toolbar. Therefore I'd use "option".
On another occasion I'd also fix the SeenThis! description in the linked article.

File: docs/ibexa_engage/ibexa_engage.md [hunk 29]
```diff
@@ -0,0 +1,85 @@
...
+To do it, select custom tag from the toolbar, and a choose campaign.
```
Comment (dabrt) [inline docs/ibexa_engage/ibexa_engage.md:83]:
```suggestion
To do it, select a custom tag from the toolbar, and choose a campaign.
```
See my comment on custom tags above.

File: docs/ibexa_engage/ibexa_engage.md [hunk 30]
```diff
@@ -0,0 +1,85 @@
...
+1\. Go to the Back Office.
+
+2\. On the left panel, go to **Ibexa Engage**.
+
+3\. On the welcome page, provide the required information:
+
+- First and last name
+- Email address
+
+4\. Accept **Terms and Conditions** checkbox, and then click **Send request**.
```
[the remark this answers is not in the document]
Comment (julitafalcondusza) [reply docs/ibexa_engage/ibexa_engage.md:23]: I'd keep "Accept".

File: docs/ibexa_engage/ibexa_engage.md (hunk 14 above)
Comment (julitafalcondusza) [reply docs/ibexa_engage/ibexa_engage.md:39]: I'd keep it singular.

File: docs/ibexa_engage/ibexa_engage.md (hunk 17 above)
Comment (juskora) [reply docs/ibexa_engage/ibexa_engage.md:46]: It's not that obvious, so I would be careful with this suggestion.

File: docs/ibexa_engage/ibexa_engage.md (hunk 21 above)
Comment (julitafalcondusza) [reply docs/ibexa_engage/ibexa_engage.md:66]: For now, I'd keep single page.

File: docs/ibexa_engage/ibexa_engage.md (hunk 24 above)
Comment (julitafalcondusza) [reply docs/ibexa_engage/ibexa_engage.md:70]: User wants to display the campaign - this is the purpose of the tools. I'd keep "display" as it makes it easier to understand the goal.

File: docs/ibexa_engage/ibexa_engage.md (hunk 28 above)
Comment (julitafalcondusza) [reply docs/ibexa_engage/ibexa_engage.md:82]: "Custom tag" - this is the feature name.

**Review by dabrt — APPROVED** [review]

**Review by mnocon — APPROVED** [review]
